pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyConnection.java

```diff
@@ -91,18 +93,33 @@ public ProxyConnection(ProxyService proxyService) {
         this.state = State.Init;
     }
 
+    @Override
+    public void channelRegistered(ChannelHandlerContext ctx) throws Exception {
+        super.channelRegistered(ctx);
+        activeConnections.inc();
```

Comment:
activeConnections is from Prometheus metrics

Reply:
I meant that instead of printing the warn log, we should have a counter exposed in the stats to track the number of rejected connections over time. The warning message can fill up the logs without being visible in metrics

Reply:
I meant that instead of printing the warn log, we should have a counter exposed in the stats to track the number of rejected connections over time. The warning message can fill up the logs without being visible in metrics

Reply:
Have put the message in debug

Reply:
Ok, though still having a counter would grant visibility on what's happening, with basically no overhead :)